.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 builds
 data
+*.lockb
```

Comment:
if we are using bun, we should have the same lock file to control for dependency versions, no?

Reply:
That's a good point. Do we want to use bun or npm going forward?
Personally I use them interchangeably in my projects more often than not (which probably isn't ideal)

Reply:
I'd prefer bun because it has better Window-Mac interchangeability in my experience, but I don't feel super strongly about this